COMSE6156-unnamed/doggo-frontend
# Take quiz

New feature:
- User can now take quiz at /quiz
- User an view profile and score at /profile

How to test:
- Because of the redirection setup, please test this branch with a local user server.
    - clone the master branch of the user server
    - contact me for the .env file
    - run `python3 app.py`
- Frontend:
    - clone or checkout the take-quiz branch  
    - `npm install`
    - npm start
    - Go to http://127.0.0.1:8080 and log in
    - Visit the profile page first to see if there is a sad doggo
    - Visit the quiz page and take a quiz and go back to the profile page to see if you have new quiz history
